lib/libutils/isoc/newlib/bcmp.c

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
SPDX and license header?

Reply:
There is no license and no copyright in the [upstream file](https://www.sourceware.org/git/gitweb.cgi?p=newlib-cygwin.git;a=blob;f=newlib/libc/string/bcmp.c;h=747f8af51d0d164fb2b5e01a89ab169ec2ac7df6;hb=HEAD) I've used.
Instead there is a big [`COPYING.NEWLIB`](https://www.sourceware.org/git/gitweb.cgi?p=newlib-cygwin.git;a=blob;f=COPYING.NEWLIB;h=0c39f9031c1efbb4daf8dbb4e6324b77edd4b2f2;hb=HEAD) file. So it seems to attribute the proper copyright and license to `bcmp.c`.

Why is it that most of our files from newlib have a BSD-3-Clause license and RedHat copyright? Coming from a different repository perhaps?

Reply:
No, it's from the same. We asked someone at Redhat for clarification at some point and as a service to those using OP-TEE we're adding the license to make it easier to FOSS in the file.

Reply:
OK, I will add a SPDX line, BSD-2-Clause is certainly applicable.
As for the copyright mention, how do we know which one in COPYING.NEWLIB applies to bcmp.c? I mean, we can't be sure RedHat wrote this. Perhaps it should be attributed to someone else. And therefore, I think we'd rather not add anything else to the file. Just the SPDX line.